index.js

```diff
@@ -192,10 +170,34 @@ window.hLoader = (function(){
         return doc.documentElement.outerHTML;
     }
 
+    /**
+     * Format HoloPort's FQDN so that everything works as gold:
+     * // is for protocol
+     * test.holo.host:4141/ is for url and port
+     * 000bundleHash000/ is for bundelHash
+     * that will give //test.holo.host:4141/000bundleHash000/
```

Comment:
@maackle This is entire url at which interceptr is querried. Is that ok?

Reply:
@peeech that's what I was thinking too! :+1:

Reply:
Merging.  If this requires changes to what the resolver returns, then let me know. :-)